code/src/components/NewThoughtPost.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export const NewThoughtPost = ({ newThought, onNewThoughtChange, onFormSubmit }) => {
+  return (
+    <form className="thought-form" onSubmit={onFormSubmit}>
+      <h1>Spread the love and share your happy thought...</h1>
+      <textarea value={newThought} onChange={onNewThoughtChange} />
+      <p className="post-length">{newThought.length} / 140</p>
+      <button
+        className="submit-btn"
+        type="submit"
+        disabled={newThought.length < 5 || newThought.length > 140}>
+        <span className="heart-span">❤︎</span> Share Happy Thought <span className="heart-span">❤︎</span>
```

Comment:
The disabled attribute was nicely used here with the minimum and max character allowed.